hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -134,50 +153,13 @@ It creates the following analysis ports to retrieve the data monitored by corres
 * kdi_fifo: For key refresh operations from OTP_CTRL
 
-The following internal data structure is used to store information about each requested memory access:
-```systemverilog
-typedef struct {
-  // 1 for writes, 0 for reads
-  bit we;
-
-  // TLUL address
-  bit [TL_AW-1:0] addr;
-
-  // Contains either the requested write data or the read response data
-  bit [TL_DW-1:0] data;
-
-  // used to mask read and write data to the requested granularity
-  bit [TL_DBW-1:0] mask;
-
-  // Tag the memory transaction with the appropriate key and nonce,
-  // so that we can keep track even if a new key is requested.
-  otp_ctrl_pkg::sram_key_t key;
-  otp_ctrl_pkg::sram_nonce_t nonce;
-
-} sram_trans_t;
-```
-
 All CSR accesses made to the SRAM_CTRL register file are tracked and predicted by the scoreboard.
 
 Verifying memory accesses is somewhat involved, and makes heavy use of the [`mem_bkdr_util`]({{< relref "hw/dv/sv/mem_bkdr_util/README.md" >}}).
 
 At a high level, whenever a TL access is made to memory, all relevant information is stored in an `sram_trans_t` struct.
 
-The scrambling RAM primitive contains a data holding register, which is used to forward unwritten write data in case a read transaction immediately follows a write transaction.
-Note that the pending write data is held no matter if the back-to-back read transaction accesses the same address in memory or not.
-This means that every read transaction will return data in 1 cycle, but a write request may not actually modify the memory until many cycles later when any pending read request has been processed.
-
-This data forwarding is modeled in the scoreboard such that we check all memory transactions in the exact same order that they are actually executed by the RAM primitive.
-
-At transactions complete, we use the memory backdoor interface to read the word in memory at the transaction's address and compare it against the requested write data or the read response data, using abyte-mask corresponding to the granularity of the memory access.
-
-We use a specific set of methods in the memory backdoor interface that utilize the memory scrambling golden model in order to provide the testbench with the correct unencrypted data for these comparisons.
-
-For example, consider a memory access that writes `1` to address `0x0`.
-Internally, the scrambling RAM primitive will map address `0x0` to some other address, say scrambled address `0x4`, and will also encrypt the data.
-In the scoreboard we detect this transaction on the TLUL bus and use the memory backdoor interface and scrambling golden model to perform a backdoor read to logical address `0x0`.
-This will access the correct scrambled location in memory, decrypt the data there, and return it to the scoreboard.
-This makes it easier to diagnose issues relating to incorrect addresses, and to data encryption or decryption.
+Refer to section "Reference models" for the SRAM memory checking.
```

Comment:
Some details about the mechanics of checking in the old text are omitted in the new text. Is that intentional?

Reply:
yes, old checking has been removed and the details are in `Reference models` section